 Remove dependency on dev-python/black used to format the generated hooks.py
 file as it's not relevant in a packaging use case.
--- a/pylib/tools/hookslib.py
+++ b/pylib/tools/hookslib.py
@@ -205,4 +205,3 @@ def write_file(path: str, hooks: list[Hook], prefix: str, suffix: str):
 
     with open(path, "wb") as file:
         file.write(code.encode("utf8"))
-    subprocess.run([sys.executable, "-m", "ruff", "format", "-q", path], check=True)
